includes/cf-descriptions/retags.md

```diff
@@ -0,0 +1,3 @@
+**Retags**<br>
+
+Retags is something you often see on public trackers, where they add their tracker name to the release name, they also could show up on some usenet indexers.
```

Comment:
```suggestion
Retags are something you often see on public trackers where the tracker or uploader adds their name to the torrent. It is also possible that it could show up on some usenet indexers.